dev/NumberBox/NumberBox.xaml

```diff
@@ -29,6 +29,7 @@
                                 <VisualState x:Name="SpinButtonsPopup">
                                     <VisualState.Setters>
                                         <Setter Target="InputBox.Style" Value="{StaticResource NumberBoxTextBoxStyle}" />
```

Comment:
Does the NumberBoxTextBoxStyle not have CornerRadius? If not, should it?

Reply:
Oh actually this is to take the NumberBox CornerRadius and apply it to TextBox. That makes sense.